pkg/frontend/querymiddleware/spin_off_subqueries_test.go

```diff
@@ -29,13 +28,11 @@ import (
 
 func TestSubquerySpinOff_Correctness(t *testing.T) {
 	var (
-		numSeries                = 1000
-		numStaleSeries           = 100
-		numConvHistograms        = 1000
-		numStaleConvHistograms   = 100
-		histogramBuckets         = []float64{1.0, 2.0, 4.0, 10.0, 100.0, math.Inf(1)}
-		numNativeHistograms      = 1000
-		numStaleNativeHistograms = 100
+		numSeries      = 1000
+		numStaleSeries = 100
+		samplesStart   = time.Now().Add(-2 * 24 * time.Hour)
+		samplesEnd     = time.Now().Add(30 * time.Minute)
```

Comment:
It's possible that `time.Now()` returns slightly different results between these two lines being evaluated - perhaps it would be safer to define `samplesEnd` in terms of `samplesStart`, or store "now" once and use it in both places.